src/test/java/api/loans/scenarios/CheckInDeclaredLostItemTest.java

```diff
@@ -0,0 +1,484 @@
+package api.loans.scenarios;
+
+import static api.support.http.CqlQuery.queryFromTemplate;
+import static api.support.matchers.JsonObjectMatcher.hasJsonPath;
+import static org.hamcrest.CoreMatchers.allOf;
+import static org.hamcrest.CoreMatchers.everyItem;
+import static org.hamcrest.CoreMatchers.hasItems;
+import static org.hamcrest.CoreMatchers.not;
+import static org.hamcrest.CoreMatchers.notNullValue;
+import static org.hamcrest.CoreMatchers.nullValue;
+import static org.hamcrest.CoreMatchers.startsWith;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+import java.util.UUID;
+
+import org.folio.circulation.domain.FeeAmount;
+import org.folio.circulation.support.http.client.IndividualResource;
+import org.hamcrest.Matcher;
+import org.joda.time.DateTime;
+import org.joda.time.DateTimeZone;
+import org.junit.Before;
+import org.junit.Test;
+
+import api.support.APITests;
+import api.support.MultipleJsonRecords;
+import api.support.builders.CheckInByBarcodeRequestBuilder;
+import api.support.builders.DeclareItemLostRequestBuilder;
+import io.vertx.core.json.JsonObject;
+
+public class CheckInDeclaredLostItemTest extends APITests {
+  private static final String LOST_ITEM_FEE = "Lost item fee";
+  private static final String LOST_ITEM_PROCESSING_FEE = "Lost item processing fee";
+  private static final String CANCELLED_ITEM_RETURNED = "Cancelled item returned";
+  private static final String REFUNDED_FULLY = "Refunded fully";
+  private static final String CREDITED_FULLY = "Credited fully";
+  private static final String LOST_ITEM_FOUND = "Lost item found";
+
+  private IndividualResource item;
+  private IndividualResource loan;
+
+  @Before
+  public void activateChargeableLostItemFeePolicy() {
+    useLostItemPolicy(lostItemFeePoliciesFixture.chargeFee().getId());
+  }
+
+  @Test
+  public void canCancelItemFeeOnly() {
```

Comment:
I prefer should in this statement. I was trying to write up some reasoning why and I realised I'm not able to succinctly describe it at the moment, it's more a sense of what reads better. What do you think?

What does cancel mean, is that what happens when a fee hasn't already been paid off at all?

```suggestion
  public void shouldCancelItemFeeOnly() {
```

Reply:
> What does cancel mean, is that what happens when a fee hasn't already been paid off at all?

Yes, if remaining amount == fee amount.

Reply:
Renamed.